xhr.bs

```diff
@@ -893,8 +893,8 @@ method must run these steps:
        <li><p>while {{XMLHttpRequest/timeout!!attribute}} attribute value is not zero.
       </ol>
 
-     <li><p>If <var>req</var>'s <a for=request>done flag</a> is unset, then
-     <a for=fetch>terminate</a> <a for=/>fetching</a> with reason <i>timeout</i>.
+     <li><p>If <var>req</var>'s <a for=request>done flag</a> is unset, then set the
+     <a>timed out flag</a>, and <a for=fetch>terminate</a> <a for=/>fetching</a>.
```

Comment:
Also no need for a comma here.

Reply:
Is the rule "use commas if there are more than two things to do"?

Reply:
Yeah, or if it's needed because it would get unwieldy somehow.